MAINTAINERS

```diff
@@ -1,12 +1,14 @@
 # Moby maintainers file
 #
-# This file describes who runs the docker/docker project and how.
-# This is a living document - if you see something out of date or missing, speak up!
+# This file describes the maintainer groups within the moby/moby project.
+# More detail on the Moby project governance is available in the
+# project/GOVERNANCE.md file found in this repository.
 #
 # It is structured to be consumable by both humans and programs.
 # To extract its contents programmatically, use any TOML-compliant
 # parser.
 #
+# TODO(estesp): This file should not necessarily depend on docker/opensource
```

Comment:
Was discussing this with @AkihiroSuda at some point; wondering if we should have a `moby/projects` (e.g.) repository that allows collecting the maintainers of all repos, and where we can share common documents

Reply:
Probably we can use TSC repo?

Reply:
I'm okay with that. Currently the repository contains the "rules" as well (perhaps we should remove that, because repository can keep separate rules), the repo is primarily used as an easy way to have contact information for all maintainers (so currently both "docker" open source and "moby"), not sure if that's a problem; if we should split, or not